App/src/main/res/anim/add_place_buttons_hide.xml

```diff
@@ -6,4 +6,5 @@
         android:fromYDelta="0%"
         android:toYDelta="100%"
         android:duration="300" />
+
```

Comment:
Nitpick: please avoid no-op changes, they make the pull request more confusing. :)